symbols.xml

```diff
@@ -219,22 +219,22 @@
         generated standingorders
 
         <global-address name='standing_orders_mix_food' value='0x00f9303d'/>
-        <global-address name='standing_orders_gather_bodies' value='0x00f9303e'/>
```

Comment:
Where did you get the new values for these? `devel/find-offsets standing_orders` should pretty much always be right, so I would be wary of static analysis that doesn't match that. (Of course, I don't know whether these are from find-offsets or not.)